logback-classic/src/main/java/ch/qos/logback/classic/Logger.java

```diff
@@ -289,11 +280,11 @@ public boolean detachAppender(Appender<ILoggingEvent> appender) {
      * Create a child of this logger by suffix, that is, the part of the name
      * extending this logger. For example, if this logger is named "x.y" and the
      * lastPart is "z", then the created child logger will be named "x.y.z".
-     * 
+     *
```

Comment:
I'd be inclined to take out unnecessary, unrelated changes to make it easier on the maintainers

Reply:
Sorry, this is caused by my IDEA setting, now I revert this.